tests/taskcluster/lib/decorate_task.js

```diff
@@ -96,12 +96,33 @@ function decorateTask(task, options) {
   output.task.scopes.push('docker-worker:image:' + IMAGE + '*');
 
   if (process.env.TREEHERDER_PROJECT && process.env.TREEHERDER_REVISION) {
+    // defaults to 'tc-treeherder' only so existing tasks need not be changed
+    var prefixes = ['tc-treeherder.'];
```

Comment:
Do we want to default to publishing results to prod treeherder when task.extra is not specified?  Perhaps we could report by default to staging and update the existing tasks to have the correct task.extra information.  I would redirect that to Kevin though.